webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/editablemapping/EditableMappingService.scala

```diff
@@ -455,16 +455,29 @@ class EditableMappingService @Inject()(
                                                                version,
                                                                parameters.agglomerateId,
                                                                remoteFallbackLayer) ?~> "getAgglomerateGraph.failed"
-      edgesToCut <- minCut(agglomerateGraph, parameters.segmentId1, parameters.segmentId2).toFox ?~> "Could not calculate min-cut on agglomerate graph."
+      edgesToCut <- minCut(agglomerateGraph, parameters.partitionOne, parameters.partitionTwo).toFox ?~> "Could not calculate min-cut on agglomerate graph."
       edgesWithPositions = annotateEdgesWithPositions(edgesToCut, agglomerateGraph)
     } yield edgesWithPositions
 
-  private def minCut(agglomerateGraph: AgglomerateGraph, segmentId1: Long, segmentId2: Long): Box[List[(Long, Long)]] =
+  private def minCut(agglomerateGraph: AgglomerateGraph,
+                     partitionOne: List[Long],
+                     partitionTwo: List[Long]): Box[List[(Long, Long)]] = {
+    val firstPartitionRootId = -1
+    val secondPartitionRootId = -2
```

Comment:
Now we also have the keyword first and second. Here I’d also vote for 1 and 2. Also these lines could use a comment. Maybe move them below the existing comment about the artificial root nodes?